samples/snippets/revoke_dataset_access_test.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .. import update_dataset_access
+from .. import revoke_dataset_access
+
+
+def test_revoke_dataset_access(capsys, dataset_id, entity_id):
+
+    update_dataset_access.update_dataset_access(dataset_id)
+    revoke_dataset_access.revoke_dataset_access(dataset_id, entity_id)
+
+    out, err = capsys.readouterr()
+    assert f"Updated dataset '{dataset_id}' with modified user permissions." in out
```

Comment:
Could you look for a message specific to "revoke"? I worry that this could catch the original update and the revoke could be failing.

Reply:
Also, let's add some additional steps to call `get_dataset` and check that the dataset access was actually updated.

Reply:
would you recommend the pytest fixture pattern to create a bigquery client that I can call this on? I'm seeing it in some other snippets but not the update_access sample. I also need to move `get_dataset` to `/snippets` as well!